xml_analyzer/data_container.py

```diff
@@ -6,21 +6,31 @@ def __init__(self):
         self.mapping_element = collections.namedtuple('property_value_mapping', ['property', 'value'])
         self.mappings = dict()
```

Comment:
Are all of these instance attributes really public, if yes then why?

Reply:
Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/6614ef2d31e5e3284ee434bbd113e446107a9606